src/query_node.h

```diff
@@ -126,8 +126,11 @@ typedef struct {
 typedef enum {
   QueryNode_Verbatim = 0x01,
   QueryNode_OverriddenInOrder = 0x02,
+  QueryNode_YieldsDistance = 0x04,
 } QueryNodeFlags;
 
+#define YIELD_DISTANCE_ATTRIBUTE "yield_distance_as"
```

Comment:
Why is it here?
We can have it directly in `QueryNode_ApplyAttribute`.

Reply:
I think it's a better practice to have hard-coded strings as a define, no?

Reply:
Either change all string to this format or follow existing formating rules.